docs/page-loaders.md.js

```diff
@@ -0,0 +1,73 @@
+process.stdout.write(`---
+keywords: server-side rendering, ssr
+---
+
+# Page loaders <a href="https://github.com/observablehq/framework/pull/1523" class="observablehq-version-badge" data-version="prerelease" title="Added in #1523"></a>
+
+Page loaders are like [data loaders](./data-loaders), but for generating (or “server-side rendering”) dynamic pages. Page loaders are programs that emit [Markdown](./markdown) to standard out, and have a double extension starting with \`.md\`, such as \`.md.js\` for a JavaScript page loader or \`.md.py\` for a Python page loader.
+
+By “baking” dynamically-generated content into static Markdown, you can further improve the performance of pages since the content exists on page load rather than waiting for JavaScript to run. You may even be able to avoid loading additional assets and JavaScript libraries.
+
+For example, to render a map of recent earthquakes into static SVG using D3:
+
+~~~js run=false
+import * as d3 from "d3-geo";
+import * as topojson from "topojson-client";
```

Comment:
this example does not work out of the box. You get
`Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'topojson-client' imported from`.

and you can't fix it (apparently?) by using `"npm:topojson-client"`, you have to `yarn add topojson-client`

Reply:
You need to `yarn add --dev d3-geo topojson-client`. WAI.